examples/ising/ising/portrayal.py

```diff
@@ -0,0 +1,15 @@
+def portray_spin(spin):
+    """
+    This function is registered with the visualization server to be called
+    each tick to indicate how to draw the cell in its current state.
+    :param cell:  the cell in the simulation
+    :return: the portrayal dictionary.
+    """
+    if spin is None:
+        raise AssertionError
+    return {
+        "marker": "s",
```

Comment:
Already using the new marker function, awesome!

I think we named it shape though: https://github.com/projectmesa/mesa/pull/2214

```suggestion
        "shape": "s",
```
Or if this intended?

Reply:
CC @rmhopkins4